### ISSUE TITLE
everything worked fine but

### ISSUE THREAD

**Issue by nithinkodadhavadee** [ubports/ubports-installer#1814]
**UBports Installer `0.8.4-beta` (deb)**
Environment: `Ubuntu 20.04.2 LTS Focal Fossa linux 5.8.0-44-generic x64 NodeJS v12.18.3`
Device: bacon
Target OS: Ubuntu Touch
Settings: `undefined`
OPEN-CUTS run: https://ubports.open-cuts.org/run/603a52e416ef7600077b1cf8
Log: https://paste.ubuntu.com/p/xWKsCbxdvD/

I'm completely to new to this and have no idea about unlocking the bootloader and all, so when the installer is done with the unlocking, I wish it would have been even more helpful if there was a message saying to restart the installation process because the installed did nothing but say unlocking the firmware even whet that was done.

**Comment by NeoTheThird** [comment]
You mean you had to close and re-open the installer for it to work after unlocking the bootloader? That is suprising, it should usually be able to resume.

**Comment by nithinkodadhavadee** [comment]
yes, I waited for quite a long time but there was no change on the screen. I switched on the developer mode on my phone after the restart to see if that was the issue but didn't see any change in the installer. So I closed the installer and started it again.

**Comment by NeoTheThird** [comment]
Hmm, interesting. We'll have to do some digging to see what's going on. Thanks for getting back to me!
